layouts/partner/aws.html

```diff
@@ -281,6 +208,115 @@ <h3 class="mt-12 mb-8 text-center">
         </section>
     </div>
 
+    <!-- Automation API -->
+    <section id="automation-api" class="container mx-auto mb-12">
+        <div class="flex flex-col lg:flex-row">
+            <div class="w-full lg:w-5/12 px-8">
+                <p class="text-3xl font-bold inline-block">Enable Everyone To Do Everything</p>
+                <p>
+                    The Automation API gives you the ability to build bespoke deployment tools that allow
+                    application developers to self-serve their infrastructure, using codified best practices.
+                    No two applications or environments are the same and you can quickly hit edges with tools
+                    trying to be one-size-fits all solution. The Automation API enables you to write tooling that fits
+                    your use cases and works exactly the way you want it to. Everything from complex workflow
+                    orchestration to opinionated full-stack application frameworks is possible with the Automation API.
+                </p>
+                <p class="my-4 mt-8 text-left w-full">
+                    <a class="inline-flex items-center text-gray-700 hover:text-gray-800 font-bold text-xl" href="{{ relref . "/blog/automation-api" }}">
+                        <span class="inline-block w-6 text-orange-300 mr-3">
+                            <svg aria-hidden="true" focusable="false" data-prefix="fad" data-icon="sparkles" class="svg-inline--fa fa-sparkles fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><g class="fa-group"><path class="fa-secondary" fill="currentColor" d="M423.16 186.58L448 127l59.58-24.84a8 8 0 0 0 0-14.32L448 63 423.16 3.42a8 8 0 0 0-14.32 0L384 63l-59.58 24.84a8 8 0 0 0 0 14.32L384 127l24.84 59.58a8 8 0 0 0 14.32 0zm-14.32 136.84L384 383l-59.58 24.84a8 8 0 0 0 0 14.32L384 447l24.84 59.58a8 8 0 0 0 14.32 0L448 447l59.58-24.84a8 8 0 0 0 0-14.32L448 383l-24.84-59.58a8 8 0 0 0-14.32 0z" opacity="0.4"></path><path class="fa-primary" fill="currentColor" d="M384 254.64a16.06 16.06 0 0 0-8.84-14.33l-112.57-56.39-56.28-112.77c-5.44-10.87-23.19-10.87-28.62 0l-56.28 112.77L8.84 240.31a16 16 0 0 0 0 28.67l112.57 56.39 56.28 112.77a16 16 0 0 0 28.62 0l56.28-112.77L375.16 269a16.07 16.07 0 0 0 8.84-14.36z"></path></g></svg>
```

Comment:
We've copied this around a bunch now, looks like. How about we pull it into a parameterized partial or something to reduce the duplication and make a more generally useful thing?

Reply:
👍